lib/Config/ServicesChecker.php

```diff
@@ -132,6 +132,23 @@ public function isMailPoetAPIKeyPendingApproval(): bool {
     return $mssActive && $mssKeyValid && $mssKeyPendingApproval;
   }
 
+  public function isUserActivelyPaying(): bool {
+    $isPremiumKeyValid = $this->isPremiumKeyValid(false);
+    $premiumSupportTier = $this->settings->get('premium.premium_key_state.data.support_tier');
+    $isUserPayingForPremium = $premiumSupportTier === 'premium';
+
+    $mssActive = Bridge::isMPSendingServiceEnabled();
+    $isMssKeyValid = $this->isMailPoetAPIKeyValid(false);
+    $mssSupportTier = $this->settings->get('mta.mailpoet_api_key_state.data.support_tier');
+    $isUserPayingForMss = $mssSupportTier === 'premium';
```

Comment:
Same as above: maybe it's better to create a method similar to `hasPremiumSupport()` in `MailPoet\Util\License\Features\Subscribers`.